src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAwaitForeachTests.cs

```diff
@@ -1802,7 +1802,7 @@ public System.Threading.Tasks.Task<bool> MoveNextAsync()
                 );
         }
 
-        [Fact]
+        [ConditionalFact(typeof(NoIOperationValidation), Reason = "https://github.com/dotnet/roslyn/issues/49267")]
```

Comment:
>https://github.com/dotnet/roslyn/issues/49267 [](start = 67, length = 45)

It looks like this issue is been actively worked on. Please coordinate @chsienki to make sure the test are in enabled state after both PRs are merged.